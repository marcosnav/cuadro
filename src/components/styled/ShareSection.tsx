import styled from 'styled-components';

const ShareSection = styled.div`
  display: flex;
  font-size: 0.83em;
  margin-top: 10px;
  justify-content: center;
  width: 300px;

  & > * {
    margin: 0 10px;
  }
`;

ShareSection.displayName = 'ShareSection';

export default ShareSection;
