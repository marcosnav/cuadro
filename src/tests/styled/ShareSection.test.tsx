import { mount } from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import ShareSection from './../../components/styled/ShareSection';

describe('<ShareSection/>', () => {
  test('renders properly', () => {
    const component = mount(<ShareSection/>);
    expect(component).toHaveStyleRule('display', 'flex');
    expect(component).toHaveStyleRule('justify-content', 'center');
    expect(component).toMatchSnapshot();
  });
});
