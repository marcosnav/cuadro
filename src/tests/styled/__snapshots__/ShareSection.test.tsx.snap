// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShareSection/> renders properly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.83em;
  margin-top: 10px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 300px;
}

.c0 > * {
  margin: 0 10px;
}

<ShareSection>
  <div
    className="c0"
  />
</ShareSection>
`;
